src/index.js

```diff
@@ -392,10 +392,6 @@ class Usabilla {
     this.email = new EmailProduct('/live', signatureFactory, this.config);
     this.apps = new AppsProduct('/live', signatureFactory, this.config);
   }
-
-  configure (options) {
-    assign(this.config, options);
-  }
 }
```

Comment:
This function is used to alter the configuration in the Usabilla class - e.g. different (mock) server. Please leave it in.
We should write an example for this..